.buildbot.sh

```diff
@@ -47,8 +47,7 @@ echo -n "1+++" | cargo run --package nimbleparse -- src/param.l src/param.y -
 cd $root/lrpar/examples/start_states
 echo -n "/* /* commented out */ */ uncommented text /* */" | cargo run --package nimbleparse -- src/comment.l src/comment.y -
 cd $root/lrpar/cttests/
-# Test wasm32-unknown-unknown under nodejs.
-wasm-pack test --node
+cargo test --target wasm32-unknown-unknown
 cargo test --target wasm32-wasip2
 cargo test
```

Comment:
These would ideally run `cargo test --target ...` just from the root rather than within this subdir.
However that doesn't seem to work, building the `tempfile` crate needed for `lrpar::ctbuilder`.

I don't have a complete understanding of what is going on but it seems like that tries to run the *codegen* process from within wasi on `wasip2`, and that runs into some issue with nightly features in the std library.

Thus currently we can only `build` the targets at top-level but only `test` them on this cttests.

Reply:
I think this was improved in [96f24cb](https://github.com/softdevteam/grmtools/pull/527/commits/96f24cb215859aaac571b7d7d69ceb311e552f65)

By disabling those tests that required tempfile.